# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/serf/hotfix-openssl-3.0.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- serf-1.3.9/buckets/ssl_buckets.c.vanilla	2021-09-15 18:23:36.009040101 +0200
+++ serf-1.3.9/buckets/ssl_buckets.c	2021-09-15 18:24:05.311038933 +0200
@@ -1325,8 +1325,7 @@
                 return 0;
             }
             else {
-                printf("OpenSSL cert error: %d %d %d\n", ERR_GET_LIB(err),
-                       ERR_GET_FUNC(err),
+                printf("OpenSSL cert error: %d %d\n", ERR_GET_LIB(err),
                        ERR_GET_REASON(err));
                 PKCS12_free(p12);
                 bio_meth_free(biom);
